src/style.css

```diff
@@ -0,0 +1,148 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.bigSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 16px;
+}
+
+.smallSearchBar::placeholder {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-size: 14px;
+}
+
+.first_form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.second_form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.bigSearchBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+  height: 70px;
+  padding-left: 62px;
+}
+
+.smallSearchBar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+  padding-bottom: 2px;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.bigLoop {
+  position: absolute;
+  left: 26px;
+  top: 25px;
+}
+
+.smallLoop {
+  position: absolute;
+  left: 13px;
+  top: 15px;
+  height: 11px;
+  width: 11px;
+}
+
+.bigSearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61,78,97,0.2);
+}
+
+.smallSearchBar:hover {
+  box-shadow: 0 3px 8px rgba(61,78,97,0.2);
+}
+
+.bigSearchBar:focus {
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+  border-radius: 3px;
+  color: #3d4e61;
+  font-weight: 700;
+}
+
+.smallSearchBar:focus {
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+  border-radius: 3px;
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search-form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-bar {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  display: block;
+  width: 100%;
+  font-weight: 300;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61,78,97,0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  padding: 23px 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search-bar-small {
+  padding: 11px 33px 10px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
This is right way to do it
I don't understand why you've written right styles here, but used duplicated classes instead of this